dpctl/tests/test_sycl_context.py

```diff
@@ -356,7 +356,7 @@ def test_context_equals():
     try:
         ctx1 = dpctl.SyclContext("gpu")
         ctx0 = dpctl.SyclContext("gpu")
-    except dpctl.SyclQueueCreationError:
+    except ValueError:
```

Comment:
This is good. I think we may as well introduce `SyclContextCreationError` and use that, for symmetry,